.travis.yml

```diff
@@ -14,15 +14,14 @@ before_install:
   # Update conda itself
   - conda update --yes conda
 install:
-  - conda create --yes -n env_name python=$PYTHON_VERSION pip nose openpyxl=1.8.2 pandas>=0.15 pyzmq flake8
-  - conda install --yes -n env_name python=$PYTHON_VERSION h5py>=2.3.1
+  # install a few of the dependencies that pip would otherwise try to install
+  # when intalling scikit-bio
+  - conda create --yes -n env_name python=$PYTHON_VERSION pip nose openpyxl=1.8.2 'pandas>=0.15' pyzmq flake8 'matplotlib>=1.1.0' 'ipython[all]==2.4.1' 'scipy>0.13.0' 'numpy>= 1.7' 'h5py>=2.3.1'
   - source activate env_name
-  - pip install coveralls cogent burrito qcli burrito-fillings ipython[all]==2.4.1
+  # upgrade click, older versions of click do not have __version__
+  - pip install -U click coveralls
   - pip install https://github.com/biocore/mustached-octo-ironman/archive/master.zip
-  - pip install https://github.com/biocore/qiime/archive/master.zip --no-deps
```

Comment:
Not really, this is removed because, as @gregcaporaso pointed out last week, the latest release version of QIIME is installed when we pip-install qiita a line below.